pkg/controller/miganalytic/miganalytics_controller.go

```diff
@@ -275,6 +287,75 @@ func (r *ReconcileMigAnalytic) analyze(analytic *migapi.MigAnalytic) error {
 	return nil
 }
 
+func (r *ReconcileMigAnalytic) analyzeExtendedPVCapacity(client compat.Client, ns *migapi.MigAnalyticNamespace) error {
+	return nil
+}
+
+func (r *ReconcileMigAnalytic) getNodeToPVMapForNS(ns *migapi.MigAnalyticNamespace) (map[string][]persistentVolumeDetails, error) {
+
+	podList := corev1.PodList{}
+	nodeToPVDetails := map[string][]persistentVolumeDetails{}
+
+	err := r.Client.List(
+		context.TODO(),
+		k8sclient.InNamespace(ns.Namespace),
+		&podList)
+
+	if err != nil {
+		return nil, liberr.Wrap(err)
```

Comment:
```suggestion
		return nodeToPVDetails, liberr.Wrap(err)
```
Better to return empty map since this is not a reference

Reply:
Done